src/pages/journal/journal.ts

```diff
@@ -7,19 +7,22 @@ import { BasePageComponent } from '../../classes/base-page';
 import { AuthenticationProvider } from '../../providers/authentication/authentication';
 import * as journalActions from './journal.actions';
 import { StoreModel } from '../../common/store.model';
-import { getTestSlots, getError, getIsLoading } from './journal.selector';
+import { getTestSlots, getError, getIsLoading, getLastRefreshed } from './journal.selector';
 import { getJournalState } from './journal.reducer';
 import { MesError } from '../../common/mes-error.model';
 import { map } from 'rxjs/operators';
 import { SlotSelectorProvider } from '../../providers/slot-selector/slot-selector';
 import { SlotComponent } from './components/slot/slot';
 import { merge } from 'rxjs/observable/merge';
 import { SlotItem } from '../../providers/slot-selector/slot-item';
+import { isNil } from 'lodash';
+import * as moment from 'moment';
 
 interface JournalPageState {
   testSlots$: Observable<SlotItem[]>,
   error$: Observable<MesError>,
   isLoading$: Observable<boolean>,
+  lastRefreshed$: Observable<Date>,
```

Comment:
Would be more accurate to call this `lastRefreshedTime$` so that we know that it refers to the actual time.

Reply:
Done.